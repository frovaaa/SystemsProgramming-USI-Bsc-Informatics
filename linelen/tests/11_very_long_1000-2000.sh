#!/bin/bash
exec "$1" 1000 2000 < tests/input1000-5002.txt
