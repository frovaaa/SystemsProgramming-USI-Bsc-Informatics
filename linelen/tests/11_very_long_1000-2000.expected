1000--------------------------------------------------------...
1001++++++++++++++++++++++++++++++++++++++++++++++++++++++++...
1002========================================================...
2000--------------------------------------------------------...
